cmd/kubelet/app/options/options.go

```diff
@@ -232,6 +232,8 @@ func AddKubeletConfigFlags(fs *pflag.FlagSet, c *componentconfig.KubeletConfigur
 	fs.MarkDeprecated("experimental-fail-swap-on", "This flag is deprecated and will be removed in future releases. please use --fail-swap-on instead.")
 
 	fs.StringVar(&c.PodManifestPath, "pod-manifest-path", c.PodManifestPath, "Path to to the directory containing pod manifest files to run, or the path to a single pod manifest file. Files starting with dots will be ignored.")
+	fs.StringVar(&c.CheckpointPath, "checkpoint-path", c.CheckpointPath, "<Warning: Alpha feature> Path to to the directory where the checkpoints are stored")
```

Comment:
`--experimental-BOOTSTRAPPING-checkpoint-path` :wink:?
We talked about scoping this down a lot and being clear about the functionality at this time.

Reply:
I'm ok with @smarterclayton's suggestion for the flag `experimental-pod-preservation-path`

Reply:
I'm changing to --bootstrap-checkpoint-path  per: ' https://github.com/kubernetes/community/pull/1241/files ' 

Putting 'experimental' or 'alpha' into names has been known to cause issues during promotion and we've moved to updating descriptions and documentation.  The name change should suffice.